src/bp/core/services/ghost/service.ts

```diff
@@ -12,7 +12,7 @@ import { VError } from 'verror'
 
 import { TYPES } from '../../types'
 
-import { GhostPendingRevisions, GhostPendingRevisionsWithContent, StorageDriver } from '.'
+import { GhostPendingRevisions, ServerGhostPendingRevisions, StorageDriver } from '.'
```

Comment:
I'm not sure whats the difference between `GhostPendingRevisions` and `ServerGhostPendingRevisions`. One is local and the other comes from the server? If that is the case I'd just prepend `Local` to `GhostPendingRevisions`.

But I know it starts to look like a Java variable name :P

Reply:
Server means server wide. Let me check again